action.yml

```
@@ -0,0 +1,39 @@
+name: sigstore-installer
+author: sigstore
+description: 'Installs sigstore tools and includes it in your path'
+branding:
+  icon: 'package'
+  color: 'blue'
+
+# This is pinned to the last major release, we have to bump it for each action version.
+inputs:
+  install-dir:
+    description: 'Where to install the cosign binary'
+    required: false
+    default: '${HOME}/.sigstore'
+  rekor-cli-version:
+    description: 'Which rekor-cli version to install'
+    required: false
+    default: 'v0.5.0'
+  cosign-version:
+    description: 'Which cosign version to install'
+    required: false
+    default: 'v1.5.2'
```

```suggestion
    default: 'v1.9.0'
```
update to latest before initial release?